Sources/App/Core/Extensions/SQLKit+functions.swift

```diff
@@ -49,20 +49,55 @@ func lower(_ arg: SQLExpression) -> SQLFunction {
     SQLFunction("LOWER", args: arg)
 }
 
-
 func unnest(_ array: SQLExpression) -> SQLFunction {
     SQLFunction("UNNEST", args: array)
 }
 
+func to_tsvector(_ array: SQLExpression) -> SQLFunction {
+    // vectors without setweight around them default to weight class 'D'
+    // which is a 0.125 multiplier on the rank
```

Comment:
If I'm reading the page you linked below correctly, that'd be 0.1, wouldn't it? From https://www.postgresql.org/docs/current/textsearch-controls.html

> The weight arrays specify how heavily to weigh each category of word, in the order:
> 
> {D-weight, C-weight, B-weight, A-weight}
>
> If no weights are provided, then these defaults are used:
>
> {0.1, 0.2, 0.4, 1.0}

Not that the detail matters here, just to make sure I'm following along!

Reply:
Yep, you've got it right. The comments in the code for ts_rank.c indicated it was the full values that I had in my comments originally, but at the top they have defines that set it down to those specific numbers.